kits/democracy/contracts/DemocracyKit.sol

```diff
@@ -15,7 +15,44 @@ contract DemocracyKit is BetaKitBase {
         public
     {}
 
-    function newToken(string name, string symbol) external returns (MiniMeToken token) {
+    function newDemoInstance(string name) external {
```

Comment:
Not a super big fan of having this available by default; this seems like it'd be better as a "demo config" that's specified and parsed off-chain by the CLI.

Reply:
Good point, let's remove it for now and figure out how to do this well in the future.